{"name": "test_add[get_add_datas0]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 2 == 3", "trace": "slef = <pytest_test.pytest_shizhan.pytest_calc.TestCalc object at 0x0000024890C81FA0>, init_calc = <pytest_test.pytest_shizhan.python_code.daic.Calculator object at 0x0000024890C81EE0>\nget_add_datas = [1, 1, 3]\n\n    @pytest.mark.run(ordder=1)\n    def test_add(slef,init_calc,get_add_datas):\n        result = init_calc.add(get_add_datas[0],get_add_datas[1])\n        expect = get_add_datas[2]\n>       assert result == expect\nE       assert 2 == 3\n\npytest_calc.py:39: AssertionError"}, "parameters": [{"name": "get_add_datas", "value": "[1, 1, 3]"}], "start": 1614321517930, "stop": 1614321517930, "uuid": "85e5fe4c-b2d2-4ca4-837a-aae8570e9519", "historyId": "2854a47cbc8b0aaf693fa49f2f97f269", "testCaseId": "7f20b7aa897260aa82433edcccd93097", "fullName": "pytest_calc.TestCalc#test_add", "labels": [{"name": "tag", "value": "run(ordder=1)"}, {"name": "suite", "value": "pytest_calc"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "LAPTOP-3I6ADUJA"}, {"name": "thread", "value": "1128-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "pytest_calc"}]}